my-app/src/components/Carusela.js

```diff
@@ -12,7 +12,7 @@ const Carusela = ({ title, cards }) => {
     infinite: true,
     speed: 500,
     slidesToShow: 5,
-    slidesToScroll: 1,
+    swipeToSlide: true,
```

Comment:
Can't believe I've missed that part. It was so annoying that it slided only 1 item at a time!!